From c5fd8c7030ccc3274c0b17be799225f7538bc78e Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 2 Jan 2018 18:34:30 +0000
Subject: [PATCH] Add cockpit tmpfile configuration

Add systemd tmpfile for creation of /etc directories required by
cockpit.
---
 cockpit.conf | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 cockpit.conf

diff --git a/cockpit.conf b/cockpit.conf
new file mode 100644
index 0000000..8606d81
--- /dev/null
+++ b/cockpit.conf
@@ -0,0 +1,3 @@
+d /etc/cockpit - root root - -
+d /etc/cockpit/machines.d - root root - -
+d /etc/cockpit/ws-certs.d - root root - -
-- 
2.15.1

